exampleCourse/questions/elementsMatrixLatexDisplay/server.py

```diff
@@ -0,0 +1,33 @@
+
+
+import random, math
+import numpy as np
+import prairielearn as pl
+import scipy.linalg as sla
+
+def generate(data):
+
+    sf = 2
+
+    # Matrix shape
+    M = 3
+
+    # Generating the orthogonal matrix U
+    # (numbers rounded with 2 decimal digits)
+    X = np.random.rand(M,M)
```

Comment:
Maybe `M, M` (and `sf + 1`, `M + 3`, `M, 1`, etc., in subsequent lines)? This is the style we often use elsewhere.